Extensions/PanelSpriteObject/panelspriteruntimeobject-pixi-renderer.js

```diff
@@ -29,6 +29,7 @@ gdjs.PanelSpriteRuntimeObjectPixiRenderer = function(runtimeObject, runtimeScene
         this._spritesContainer.addChild(this._borderSprites[i]);
     }
 
+    this._opacity = this._spritesContainer.alpha;
```

Comment:
nitpicking: let's keep the name "opacity" for the thing that goes between 0 and 255. As this variable will contain something that is equivalent to alpha in pixi, which goes between 0 and 1, let's call it `this._alpha`.